test/basic.tests.js

```diff
@@ -42,6 +42,38 @@ contract('LiquidityBridgeContract', async accounts => {
         });
         expect(utils.LP_COLLATERAL).to.be.a.bignumber.eq(registered);
     });
+    
+    it('Should fail on register if is already registered', async () => {
+        let currAddr = accounts[7];
+
+        let tx = await instance.register({from: currAddr, value : utils.LP_COLLATERAL});
+        
+        truffleAssertions.eventEmitted(tx, "Register", { 
+            from: currAddr
+        });
+        
+        await truffleAssertions.fails(instance.register({from: currAddr, value : utils.LP_COLLATERAL}));
+    });
+
+    it('Should fail on register if not deposit the minimum collateral', async () => {
+        let currAddr = accounts[5];
+
+        await truffleAssertions.fails(instance.register({from: currAddr, value : web3.utils.toBN(0)}));
+    });
+
+    it('Should fail on register if not deposit the minimum collateral', async () => {
+        let currAddr = accounts[6];
+
+        const tx = await instance.register({from: currAddr, value : utils.LP_COLLATERAL});
+        truffleAssertions.eventEmitted(tx, "Register", { 
+            from: currAddr
+        });
+        const resignTx = await instance.resign({ from: currAddr });
+        truffleAssertions.eventEmitted(resignTx, "Resigned", { 
+            from: currAddr
+        });
+        await truffleAssertions.fails(instance.register({from: currAddr, value : utils.LP_COLLATERAL}));
```

Comment:
do you think we could use `truffleAssertions.reverts()` with an expected revert message in these tests?